modules/imaging_uploader/php/imaging_uploader.class.inc

```diff
@@ -541,7 +541,7 @@ class Imaging_Uploader extends \NDB_Menu_Filter_Form
             'UploadedBy'               => $user_name,
             'UploadDate'               => $date,
             'UploadLocation'           => $this->uploaded_file_path,
-            'SessionID'                => $sessionid ?? '',
+            'SessionID'                => $sessionid ?? null,
```

Comment:
@cmadjar @nicolasbrossard Can you comment on whether this case should occur? Is it OK for this column to be null after uploading or should it always be attached to a session ID?

Reply:
@johnsaigle You can upload a scan that is not tied to *any* session. When the pipeline processes that scan, it will create a session for you if you have `createVisitLabel` set to `1` in your `prod` file. So yeah, `$sessionid` can be `null`.